### PR TITLE
Update dataset.json5

### PR DESCRIPTION
**Pull request by ahcyip**

### CODE REVIEW

File: datasets/modeled/transport/dataset.json5 [hunk 1]
```diff
@@ -1,59 +1,94 @@
 {
-  dataset_id: "tempo_conus_2022",
+  dataset_id: "decarb_2023_transport",
   dataset_type: "modeled",
-  data_source: "tempo",
-  sector_description: "transportation",
-  description: "CONUS 2022 TEMPO county-level results for PEV charging.",
+  data_source: "tempo evix polaris",
+  sector_description: "electric load associated with light, medium, and heavy-duty road transportation",
+  description: "DECARB 6 results from TEMPO, EVI-X, and POLARIS (Nov 2023) with profiles using 2018 as the weather year",
```
Comment (elainethale) [inline datasets/modeled/transport/dataset.json5:6]: nit: "DECARB Task 6"

File: datasets/modeled/transport/dataset.json5 [hunk 2]
```diff
@@ -1,59 +1,94 @@
...
   origin_creator: "Arthur Yip",
   origin_organization: "NREL",
   origin_contributors: [
     "Arthur Yip",
+    "Brennan Borlaug"
```
Comment (elainethale) [inline datasets/modeled/transport/dataset.json5:11]: There are probably more collaborators that should be credited? TEMPO and POLARIS teams???

File: datasets/modeled/transport/dataset.json5 [hunk 3]
```diff
@@ -1,59 +1,94 @@
...
   ],
-  origin_project: "",
-  origin_date: "",
-  origin_version: "",
-  source: "",
+  origin_project: "DECARB 6",
```
Comment (elainethale) [inline datasets/modeled/transport/dataset.json5:13]: DECARB Year 1, Task 6

File: datasets/modeled/transport/dataset.json5 [hunk 4]
```diff
@@ -1,59 +1,94 @@
...
+  origin_date: "2023-11-30",
+  origin_version: "0.1.0",
+  source: "TEMPO, EVI-X, and POLARIS simulations for DECARB 6",
```
Comment (elainethale) [inline datasets/modeled/transport/dataset.json5:16]: You could add software versioning details here if those are available. For an example: https://github.com/dsgrid/dsgrid-project-DECARB/blob/main/datasets/modeled/buildings/dataset.json5#L21

File: datasets/modeled/transport/dataset.json5 [hunk 5]
```diff
@@ -1,59 +1,94 @@
...
   data_classification: "low",
   tags: [
-    "transportation",
-    "tempo",
+    "TEMPO",
+    "EVIX",
+    "POLARIS",
+    "EV",
```
Comment (elainethale) [inline datasets/modeled/transport/dataset.json5:22]: nit: "electric vehicles"?

File: datasets/modeled/transport/dataset.json5 [hunk 6]
```diff
@@ -1,59 +1,94 @@
...
+    "decarb",
   ],
+  data_schema_type: "standard",
```
Comment (elainethale) [inline datasets/modeled/transport/dataset.json5:25]: Obsolete?

File: datasets/modeled/transport/dataset.json5 [hunk 7]
```diff
@@ -1,59 +1,94 @@
...
+    { 
+     "class": "County",
       type: "geography",
-      name: "ACS County 2018",
+      name: "US counties 2020 L48",
       display_name: "County",
       file: "dimensions/counties.csv",
-      description: "American Community Survey US counties, 2018.",
+      module: "dsgrid.dimension.standard",
+      description: "US counties for L48, Census 2020 vintage, no territories",
```
Comment (elainethale) [inline datasets/modeled/transport/dataset.json5:45]: If the list is exactly the same as the project, I'd directly reuse all that information.

**Review by elainethale — CHANGES_REQUESTED** [review]

File: datasets/modeled/transport/dataset.json5 (hunk 6 above)
Comment (daniel-thom) [reply datasets/modeled/transport/dataset.json5:25]: This line must be deleted.

File: datasets/modeled/transport/dataset.json5 (hunk 7 above)
Comment (daniel-thom) [reply datasets/modeled/transport/dataset.json5:45]: The data includes some (not all) counties from Alaska and Hawaii, so this name and description should be changed.

File: datasets/modeled/transport/dataset.json5 (hunk 4 above)
Comment (ahcyip) [reply datasets/modeled/transport/dataset.json5:16]: not available

**Review by elainethale — APPROVED** [review]
This looks good to me now, assuming the associated county data matches the description (lower 48 states only).

**Pull request merged**